# the name of the objects refer to the list of the objects and not to the gazebo model IDs
# for example, name of objects are: beer, sprite, coke_can

string name_of_the_object_to_change
string name_of_the_object_to_spawn

---
#response
bool success
string message
